manage.py

```diff
@@ -152,6 +163,10 @@ def encrypted_carbon_black_api_token(self, value: str):
             )
         self._config['encrypted_carbon_black_api_token'] = value
 
+    @property
+    def force_destroy(self) -> str:
+        return self._config['force_destroy']
```

Comment:
is the `'force_destroy'` key getting set in the config? I'm not seeing it 🤔

Reply:
Yes, it's in the `terraform.tfvars`, which is what gets loaded into the `Config` object